luxonis_ml/data/parsers/luxonis_parser.py

```diff
@@ -114,16 +114,16 @@ def __init__(
         self.dataset = self.dataset_constructor(dataset_name=dataset_name, **kwargs)
         self.parser = self.parsers[self.dataset_type](self.dataset)
 
-    def parse(self, **kwargs) -> BaseDataset:
```

Comment:
I changed this from `LuxonisDataset` -> `BaseDataset` previously since if we use a custom `dataset_plugin` we could have another class than `LuxonisDataset`

Reply:
Gotcha, that makes sense

Reply:
In that case we'd need to add `make_splits` to the `BaseDataset` interface.

Reply:
> In that case we'd need to add make_splits to the BaseDataset interface.

I think we can do that. It was probably an oversight that it wasn't already there